data-model/src/main/scala/za/co/absa/enceladus/model/conformanceRule/package.scala

```diff
@@ -121,6 +122,14 @@ package object conformanceRule {
     override def withUpdatedOrder(newOrder: Int): ConformanceRule = copy(order = newOrder)
   }
 
+  case class FillNullsConformanceRule(order: Int,
+                                      controlCheckpoint: Boolean,
+                                      outputColumn: String,
+                                      inputColumn: String,
```

Comment:
It seems a little confusing that even though many of the specific conformance rules have some common parameters, their order varies. One time, it is `{order, outputColumn, controlCheckpoint, ...}`, another time it is `{order, controlCheckpoint, outputColumn, ...}`. Maybe the API would be more usable if it followed one suit.

Reply:
I copied one and enhanced it. Did not think about it. I can fix it.